src/main/java/com/revature/validation/exceptions/NotFoundException.java

```diff
@@ -0,0 +1,15 @@
+package com.revature.validation.exceptions;
+
+/**
+ * Exception used for when and entry is not found in the database and a 404 Not Found
+ * http status code is desired
+ */
+public class NotFoundException extends RuntimeException {
+
+    public NotFoundException(){}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1186)